### DISCUSSION TITLE
现在好像没有chat with pdf的功能

### DISCUSSION THREAD

**Discussion by seanxuu** [ChatGPTNextWeb/NextChat#4348] [Ideas]
有兴趣做吗？

**Comment by lloydzhou** [comment]
1. v2.15.0版本支持将外部api包装成插件使用。
2. 所以，支持包装docx的服务或者fastgpt的知识库api作为插件使用
3. 这里有一个示例 https://github.com/ChatGPTNextWeb/NextChat-Awesome-Plugins/tree/main/plugins/fastgpt
